.github/CONTRIBUTING.md

```diff
@@ -13,7 +13,7 @@ get involved:
   from the `master` branch and create a 
   [pull request](https://help.github.com/articles/about-pull-requests) for your change.
   Note that new notebooks should be placed in the relevant part of the 
-  [Community](./community/) section.
+  [Community](https://github.com/qiskit-community/qiskit-community-tutorials) section.
```

Comment:
I still think the text in this block is wrong. It's talking about forking _this_ repo and creating a pull request in _this_ repo, which isn't what you'd do. So maybe this section should just say:

```
* **Notebooks**: If you would like to contribute a notebook, please do it in the [Qiskit Community Tutorials](https://github.com/qiskit-community/qiskit-community-tutorials) project.
```

Also, note that the same docs issue exists in that repo (it's probably a fork of this one):

https://github.com/qiskit-community/qiskit-community-tutorials/blob/master/.github/CONTRIBUTING.md

Reply:
Oh, ok. So people can contribute only at qiskit-community-tutorials and they should be directed there to open PRs?

Reply:
People are free to contribute tutorials here, although the selection process is a bit more picky here than in the community.

Reply:
Oh ok cool. Is the new commit appropriate?